mgmtsystem_action/mgmtsystem_action.py

```diff
@@ -45,6 +45,18 @@ class mgmtsystem_action(models.Model):
     system_id = fields.Many2one('mgmtsystem.system', 'System')
     company_id = fields.Many2one('res.company', 'System', default=own_company)
 
+    stage_id = fields.Many2one(
+        'mgmtsystem.action.stage',
+        'Stage',
+        default=lambda self: self.get_default_stage()
+    )
+
+    @api.model
+    def get_default_stage(self):
+        return self.env['mgmtsystem.action.stage'].search([
+            ('is_starting', '=', True)
+        ]).id
```

Comment:
I could some changes myself: 
I would like to add Stages, and with that remove a lot of code that implemented a hardcoded process flow.
Also, do you actually use wkf workflows? If you let me I'm tempted to drop them also.